src/bindings/node/node_osrm.cpp

```diff
@@ -0,0 +1,507 @@
+#include "osrm/engine_config.hpp"
```

Comment:
This was me being sneaky and opening up the door for more bindings :)

Also give https://github.com/junegunn/fzf a try (or ctrl+p)